runtimes/talos-libgdx/src/main/java/com/talosvfx/talos/runtime/assets/AtlasAssetProvider.java

```diff
@@ -51,4 +51,9 @@ public ShaderDescriptor findAsset(String assetName) {
             }
         });
     }
+
+    @Override
+    public <T> GameAsset<T> findGameAsset(String assetName, Class<T> clazz) {
+        return null;
```

Comment:
Throw not implemented exception if you are going to leave it unimplemented. We would also need a task to actually implement this for runtime if you arent doing it in the scope of this task